ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/api/KeyValueParticipantState.scala

```diff
@@ -40,15 +40,22 @@ class KeyValueParticipantState(
     reader: LedgerReader,
     writer: LedgerWriter,
     metrics: Metrics,
+    apiDeduplicationEnabled: Boolean = false,
```

Comment:
Can't we just hard-code this to `true` for now?

Reply:
Done